pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -154,6 +178,20 @@ func (r *Runtime) GlobalParamDescs() params.ParamDescs {
 		}...)
 		return p
 	case ConnectionModeKubernetesProxy:
+		p.Add(params.ParamDescs{
+			{
+				Key:            ParamConnectionMethod,
+				Description:    "Method to use to connect to the gadget service; either tcp or unix-socket (deprecated)",
```

Comment:
unix-socket -> unix?

Also, perhaps this is better to avoid to ... to ... to (ditto for ParamGadgetServiceTCPPort)

```suggestion
				Description:    "Method used to connect to the gadget service; either tcp or unix-socket (deprecated)",
```